openldap/server.sls

```diff
@@ -33,6 +34,8 @@ slapd_service:
   service.running:
     - name: {{ openldap.service }}
     - enable: True
+    - watch:
+      - file: {{ openldap.server_defaults }}
```

Comment:
Either the state ID should remain as `{{ openldap.server_defaults }}` above, or this line needs to be changed to:

```suggestion
      - file: slapd_conf
```

Reply:
@myii  Ohh.. Excuse me, didn't noticed this because I hadn't any  error of this kind. Maybe, it just works with the `name` too.